src/widget.js

```diff
@@ -345,6 +347,11 @@ Widget.prototype = {
     this.closed = false;
     this.rsWidget.classList.remove('rs-closed');
     this.shouldCloseWhenSyncDone = false; // prevent auto-closing when user opened the widget
+
+    let selected = document.querySelector('.rs-box.rs-selected');
```

Comment:
I find this name to be very misleading. Selected means having selected something from an option. But this seems to not be the case for this element. I think this is about which one is visible or not, isn't it?